app/assets/stylesheets/forms.scss

```diff
@@ -0,0 +1,55 @@
+// .trans {
+//   background: rgba(0, 0, 0, 0.5);
+//   margin-top: 2rem;
+//   color: $white !important;
+//
+//   label {
+//     color: $white !important;
+//   }
+//
+//   .report-title h1 {
+//     color: $white !important;
+//   }
+// }
+
+// label {
+//   color: $white !important;
+// }
+
+.trans {
+  background: rgba(0, 0, 0, 0.5);
+  border-radius: 2px;
+  box-shadow: 0 1px 3px rgba(0,0,0,0.12), 0 1px 2px rgba(0,0,0,0.24);
+  margin-bottom: 2rem;
+  margin-top: 2rem;
+  min-height: 300px;
+  padding: 15px;
+  transition: all 0.2s ease-in-out;
+
+  &:hover {
+    box-shadow: 0 10px 20px rgba(0,0,0,0.19), 0 6px 6px rgba(0,0,0,0.23);
```

Comment:
Color literals like `rgba(0, 0, 0, 0.19)` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `rgba(0, 0, 0, 0.23)` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Commas in function arguments should be followed by a single space